Lecture-security.md

```diff
@@ -181,6 +182,15 @@ to automatically use only HTTPS, which prevents hackers from performing this
 sort of man-in-the-middle attack.
 ```
 
+### Symantec ISTR (2019)
+
+The [Internet Security Threat Report](https://www.symantec.com/content/dam/symantec/docs/reports/istr-24-2019-en.pdf) (ISTR) published by Symantec outlines the threat landscape as seen in early 2019. They determine the trends using telemetry collected from over 123 Million sensors that monitor activities of over 300,000 organizations that use Symantec’s protection. We look at a few relevant trends:
+
+1. In general, web attacks on endpoints (host machines) have increased by 56%. Spear phishing is the most prevalent infection vector used by the attackers, where they are more likely to use malicious attachments rather than malicious URLs. Nearly half of the malicious email attachments are Microsoft Office files, which are disguised either as invoices or email delivery-failure notifications.
+2. Mobile security remains a major concern: One in 36 mobile phones has a high-risk application installed. Additionally, ransomware attacks on mobile phones have increased by 33%. _Ransomware_ is a type of financial malware that makes important files on a system inaccessible by encrypting them, unless a ransom is paid.  
+3. _Formjacking_ is one of the leading threats faced by e-commerce sites: _Formjacking_ is the use of malicious JavaScript code to steal information from payment forms on the checkout pages of e-commerce sites. It has been used to compromise 4800 websites, on average, each month.
```

Comment:
This 4800 websites are attacked number is oddly small and specific. Is that 4800 websites of the entire web? Or in their sample? If we don't know the total size of the web being investigated here, this number is not too meaningful. Can that be converted somehow to percentages of e-commerce sites?

Reply:
In the report they state that they detected and blocked 3.7 Million formjacking attacks in 2018, with most of the attacks happening in the last quarter. (They probably detect this from the 1.5 Billion web requests they see each day). The average attacks per month is this 4800 number. 
I cannot find the statistics for the total ecommerce sites in 2018. I also don't know how much of the global web Symantec covers.